src/server/src/main/java/com/spotify/reaper/resources/view/NodesStatus.java

```diff
@@ -66,75 +89,76 @@ private GossipInfo parseEndpointStatesString(String sourceNode, String allEndpoi
       Optional<String> releaseVersion = Optional.absent();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable "releaseVersion". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1854)